scripts/update-report.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -euo pipefail
+set -eo pipefail
```

Comment:
any reason to yank `u` here?

Reply:
this was removed when I was adding some code to access pipeline variables in one of the iteration, I can put it back.

Script will fail if we ever try to access pipelines variables because it will always be unset
```
variable=$(SOME_VARIABLE)
```

Reply:
got it - usually desirable behavior, but I can see how it could be a hinderance in this case.